optimism/src/keccak/column.rs

```diff
@@ -34,7 +33,7 @@ pub enum KeccakColumn {
     FlagsBytes(usize),                        // 136 boolean values
     PadSuffix(usize),                         // 5 values with padding suffix
     RoundConstants(usize),                    // Round constants
-    ThetaStateA(usize, usize, usize),         // Round Curr[0..100)
+    Input(usize),                             // Curr[0..100) either ThetaStateA or SpongeOldState
```

Comment:
Nit: we will need to merge the Keccak RFC at the end. Only a personal taste: I think updating the RFC while we're updating the code might be good.

Reply:
Agree